chains/evm/calls/transactor/transact.go

```diff
@@ -83,3 +89,8 @@ func (t *signAndSendTransactor) Transact(to *common.Address, data []byte, opts T
 	}
 	return &h, nil
 }
+
+func (t *prepareTransactor) Transact(to *common.Address, data []byte, opts TransactOptions) (*common.Hash, error) {
+	log.Info().Msgf("Command calldata: %+v", data)
```

Comment:
Please take a look at the issue in more time. And think what is the purpose of this task, and HOW printed data will be used